content/client-lib-development-guide/features.textile

```diff
@@ -508,6 +548,16 @@ h4. Capability - *API not agreed yet*
 * @(TC1)@ This type represents a capability for a key or token
 * @(TC2)@ For now a string representation of the JSON will suffice wherever @capability@ is used
 
+h4. ConnectionDetails
+
+* @(CD1)@ Connection details are optionally passed to the client library in the @CONNECTED@ @ProtocolMessage#connectionDetails@ attribute to inform the client about any constraints it should adhere to, and provide additional metadata about the connection. For example, if a request is made to publish a message that exceeds the @maxMessageSize@, the client library can reject the message immediately before publishing it to Ably
```

Comment:
suggest comma after 'immediately' (currently misparsable as "the client library can reject the message, immediately before publishing it")